app/src/main/java/com/yml/womensafety/navigationdrawer/contacts/ContactsFragment.kt

```diff
@@ -1,62 +1,41 @@
 package com.yml.womensafety.navigationdrawer.contacts
 
 import android.os.Bundle
-import androidx.fragment.app.Fragment
 import android.view.View
 import android.widget.Toast
-import com.google.firebase.auth.FirebaseAuth
+import androidx.fragment.app.Fragment
 import com.google.firebase.database.*
+import com.yml.womensafety.FirebaseApplication
 import com.yml.womensafety.R
 import kotlinx.android.synthetic.main.fragment_contacts.*
 
 class ContactsFragment : Fragment(R.layout.fragment_contacts) {
-    private lateinit var firebaseAuth: FirebaseAuth
-    private lateinit var ref: DatabaseReference
+
+    private lateinit var firebaseApplication: FirebaseApplication
+    private lateinit var databaseReference: DatabaseReference
```

Comment:
We should have the firebase and database in view, as we are following mvvm, we should have it is repo layer and use interfaces to use the